contracts/vesting/src/contract.rs

```diff
@@ -277,9 +282,18 @@ fn try_create_periodic_vesting_account(
     env: Env,
     deps: DepsMut,
 ) -> Result<Response, ContractError> {
-    if info.sender != ADMIN_ADDRESS {
-        return Err(ContractError::NotAdmin(info.sender.as_str().to_string()));
+    cfg_if::cfg_if! {
+        if #[cfg(test)] {
```

Comment:
This doesn't look very nice, putting test code in the same place with production code. Isn't it possible to get the sender address in the test code? Why would we need this here?

Reply:
Admin address is set at init, I can probably mock that, have it be nicer, good call